spec/swagger_v2/api_swagger_v2_response_with_examples_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'response with examples' do
+  include_context "#{MODEL_PARSER} swagger example"
+
+  before :all do
+    module TheApi
+      class ResponseApiExamples < Grape::API
+        format :json
+
+        desc 'This returns examples' do
+          success model: Entities::UseResponse, examples: { 'application/json' => { description: 'Names list', items: [{ id: '123', name: 'John' }] } }
+          failure [[404, 'NotFound', Entities::ApiError, { 'application/json' => { code: 404, message: 'Not found' } }]]
```

Comment:
question: is it also possible to use the explicit notation as hash, mean …
```ruby
failure: [
  {
    code: 400,
    message: 'NotFound',
    model: Entities::ApiError,
    examples:  { 'application/json' => { code: 404, message: 'Not found' }
  }
]
```

Reply:
Yes, added examples and docs for that notation

https://github.com/ruby-grape/grape-swagger/pull/675/commits/b930431844f211e7a3bc583ad4798d7ffe19eecc#diff-e92729f9b6df796f94002a66a1e5b640R21